src/routes/api/configured/api-configured-post.ts

```diff
@@ -0,0 +1,45 @@
+import { Request, Response } from "express";
+import { Subscription } from "models/subscription";
+import { getLogger } from "config/logger";
+import { saveConfiguredAppProperties } from "utils/app-properties-utils";
+
+const MAX_INSTALLATIONS_BATCH = 50;
+/**
+ * This will update JIRA APP Properties for the GH4J installation to set isConfigured;
+ */
+export const ApiConfiguredPost = async (req: Request, res: Response): Promise<void> => {
+	const installationIds = req.body.installationIds as number[];
+
+	if (installationIds?.length === 0) {
+		res.status(400);
+		res.send("please provide installation ids to update!");
+		return;
+	}
+	// Upper limit of installationIds per call
+	if (installationIds.length > MAX_INSTALLATIONS_BATCH) {
+		res.status(400);
+		res.send(`Calm down Cowboy, keep it under ${MAX_INSTALLATIONS_BATCH} at a time!`);
+		return;
+	}
+
+	const logger = getLogger("api-sync-configured");
+	const tasks = installationIds.map(async installationId => {
+		// need to confirm that passed in value is configured. Existing on subscription table satisfies this.
+		const subscription = await Subscription.findOneForGitHubInstallationId(installationId, undefined);
```

Comment:
This is only gonna fetch the list of GitHub cloud subscriptions, what about the GHE ones?

Reply:
Correct, I'm in two minds about GHE, will move this chat to slack cuz I don't think we need to include them for the test.

To work with GHE, it would be a call per GitHubAppId or change the data struct of the installations = [] to installations = [{installationId: x, githubAppId: y}], wither way was adding more complexity that didn't give enough benefit IMO. But .. to slack!! and you can decide whether you want them or not